### DISCUSSION TITLE
Provide the pre-compiled package on aarch64

### DISCUSSION THREAD

**Discussion by wqh17101** [open-mmlab/mmcv#2334] [General]
**Describe the feature**
Provide the pre-compiled package on aarch64
**Motivation**
I want to run mmcv on aarch64

**Related resources**
If there is an official code release or third-party implementations, please also provide the information here, which would be very helpful.

**Additional context**
If you would like to implement the feature and create a PR, please leave a comment here and that would be much appreciated.

**Comment by zhouzaida** [comment]
Thanks for your feedback, we'll look into it and see if we can compile it in CI.
